### DISCUSSION TITLE
SMS options

### DISCUSSION THREAD

**Discussion by aemadrid-twx** [fonoster/yaps#330] [General]
Are you working on SMS support? Not as interested in voice as in SMS messaging and wondering what providers you recommend.

**Comment by psanders** [comment]
@aemadrid-twx 

SMS is within the scope of Fonoster and scheduled for [later this year](https://github.com/orgs/fonoster/projects/9). 

Do you care to share your use case? Are you sending and receiving SMS or just sending?

**Comment by aemadrid-twx** [comment]
I've used Twilio on a few different apps but only to send SMS messages. Wondering what your plans are on supporting SMS. I'm guessing you still need to use some low-level providers to actually send the SMS message and curious what providers you've found reliable and low-cost.

**Reply by psanders** [reply]
I am not sure about pricing. However, for the purpose of testing, I use VoIP MS.

**Comment by enewbury** [comment]
I would also love to use SMS. In my case I would be sending and receiving.

**Reply by psanders** [reply]
Thank you for the feedback.

**Comment by rkmullapudi** [comment]
Hello @psanders great story of yours in Github > YouTube video and thanks for building this for the community. I´ m interested in sending SMS code during user registrations in my application. Please let me know if this is available by now already.

**Reply by psanders** [reply]
Hey @rkmullapudi,

Thanks for reaching out. Unfortunately, this project is currently inactive for lack of funds.
